src/renderer/editor/properties/properties.ts

```diff
@@ -160,6 +162,17 @@ export class Properties extends EventEmitter {
       }
     });
 
+    //adding tooltip to icon.
+    
+    document.addEventListener('DOMContentLoaded', () => {
+      const iconButtons = document.querySelectorAll('[data-bs-toggle="tooltip"]');
+      iconButtons.forEach((button) => {
+          new Tooltip(button, {
+              delay: { show: 500, hide: 0 }, // Adjust delay as needed
+          });
+      });
+    });
```

Comment:
Doing this here cannot work either as the icons are loaded asynchonously, after the initial DOM has been loaded. You should rather do a `new Tooltip(iconDiv, { delay: { show: 500, hide: 0 }});` right after creating the iconDiv.

Reply:
okay i did this right after the icons are created. I cant get it what should i be exactly doing in the document addeventlistener to show the tooltip correctly